settings.sample.yaml

```diff
@@ -37,8 +37,6 @@ AUTO_PI_API_URL:
 DOCUMENTS_AWS_ACCESS_KEY_ID: test
 DOCUMENTS_AWS_SECRET_ACCESS_KEY: test
 DOCUMENTS_AWS_ENDPOINT: http://localhost:4566
-NFT_AWS_ACCESS_KEY_ID: test
```

Comment:
Not used anymore?

Reply:
Already exists https://github.com/DIMO-Network/devices-api/pull/214/files/e4ffeaa230cc75decdb078e9b88fbf3a90605569#diff-d82ffc9c00e6095149349dcf6b922139639ba720e3b77afcab0e843f4dac4539L25

Reply:
Works for me.